src/main/scala/no/ndla/articleapi/service/converters/HTMLCleaner.scala

```diff
@@ -19,7 +19,7 @@ trait HTMLCleaner {
   val htmlCleaner: HTMLCleaner
 
   class HTMLCleaner extends ConverterModule with LazyLogging {
-    private def NBSP = "\u00a0" // \u00a0 is the unicode representation of &nbsp;
+    private def NBSP = "&#xa0;" // jsoup unicode representation of &nbsp;
```

Comment:
Nei, det er egentlig en litt misvisende kommentar, det er bare en [notasjon](http://htmlhelp.com/reference/html40/entities/) jsoup bruker for å representere nbsp. Notasjonen er standard